hrms/hr/doctype/attendance_request/attendance_request.json

```diff
@@ -117,11 +118,17 @@
    "options": "Attendance Request",
    "print_hide": 1,
    "read_only": 1
+  },
+  {
+   "fieldname": "shift",
+   "fieldtype": "Link",
+   "label": "Shift",
+   "options": "Shift Type"
```

Comment:
So, the attendance request feature is also used to overwrite existing attendance **status** if the employee wants to correct it:
https://frappehr.com/docs/v14/en/attendance-request#3-1-overwrite-an-existing-attendance-record

But overwriting the shift in attendance is not really safe/clean since it has working hours and other info of the existing shift. To clarify this, let's also add a description for this field:

Note: Shift will not be overwritten in existing attendance records